src/ui/page_selector.rs

```diff
@@ -29,50 +30,55 @@ impl PageSelector {
                 button_state: button::State::new(),
                 page_type: PageType::View,
             },
+            PageButton {
+                label_text: "Dashboard".to_string(),
+                button_state: button::State::new(),
+                page_type: PageType::Dashboard,
+            },
+            PageButton {
+                label_text: "Feed".to_string(),
+                button_state: button::State::new(),
+                page_type: PageType::Feed,
+            },
+            PageButton {
+                label_text: "Site".to_string(),
+                button_state: button::State::new(),
+                page_type: PageType::Site,
+            },
+            PageButton {
+                label_text: "Settings".to_string(),
+                button_state: button::State::new(),
+                page_type: PageType::Settings,
+            },
+            PageButton {
+                label_text: "Testing".to_string(),
+                button_state: button::State::new(),
+                page_type: PageType::Testing,
+            },
         ];
 
-        if std::env::var("RUST_LOG").unwrap_or_default() == "fuzzr" {
-            let hidden_buttons = vec![
-                PageButton {
-                    label_text: "Dashboard".to_string(),
-                    button_state: button::State::new(),
-                    page_type: PageType::Dashboard,
-                },
-                PageButton {
-                    label_text: "Feed".to_string(),
-                    button_state: button::State::new(),
-                    page_type: PageType::Feed,
-                },
-                PageButton {
-                    label_text: "Site".to_string(),
-                    button_state: button::State::new(),
-                    page_type: PageType::Site,
-                },
-                PageButton {
-                    label_text: "Settings".to_string(),
-                    button_state: button::State::new(),
-                    page_type: PageType::Settings,
-                },
-                PageButton {
-                    label_text: "Testing".to_string(),
-                    button_state: button::State::new(),
-                    page_type: PageType::Testing,
-                },
-            ];
-
-            buttons.extend(hidden_buttons);
-        }
+        // if std::env::var("RUST_LOG").unwrap_or_default() == "fuzzr" {
+        //     buttons.extend(hidden_buttons);
+        // }
 
         PageSelector {
             buttons,
             active_page: PageType::Publish,
+            disabled_pages: vec![
+                PageType::Dashboard,
+                PageType::Feed,
+                PageType::Settings,
+                PageType::Site,
+                PageType::Testing,
+            ],
```

Comment:
I love this approach. That said, do you think we should still have the Testing page? We could probably just get rid of it.

Reply:
Agreed, it can go.